### ISSUE TITLE
importing phevaluator throws TypeError: 'type' object is not subscriptable

### ISSUE THREAD

**Issue by chris-jackson7** [HenryRLee/PokerHandEvaluator#74]
I've been using phevaluator for the past few weeks. Recently I started using it on Kaggle and got the error mentioned while importing it. However, my local import still worked fine. Then I made some changes to my python interpreter and now it's broken on PyCharm too. 

The traceback is:
from phevaluator import evaluate_cards
\__init\__.py > from . import hash as hash_    # FIXME: \`hash\` collides to built-in function
hash.py > def hash_quinary(quinary: list[int], num_cards: int) -> int:
TypeError: 'type' object is not subscriptable

The FIXME note makes me think this is a known problem, but I don't see any mention of it on this github page. Can anyone confirm if this is something that needs fixing? I'm not entirely sure what the issue is. Thanks!

**Comment by HenryRLee** [comment]
It seems to me that FIXME note is not related. Can you try execute the following lines in your interpreter, and see if it throws the same error?
```python
def hash_quinary(quinary: list[int], num_cards: int) -> int:
    return 0
```

**Comment by chris-jackson7** [comment]
Just running that function doesn't throw an error. It returns 0 as we'd expect.

**Comment by HenryRLee** [comment]
Ok, can you also try this?
```python
DP = [[]]
def hash_quinary(quinary: list[int], num_cards: int) -> int:
    sum_numb = 0
    length = len(quinary)
    for rank, cnt in enumerate(quinary):
        if cnt:
            sum_numb += DP[cnt][length - rank - 1][num_cards]
            num_cards -= cnt
    return sum_numb
```

**Comment by chris-jackson7** [comment]
If I give inputs to hash_quinary then DP is indexed our of its range. If I just define the function it runs without errors though.

**Comment by HenryRLee** [comment]
Thanks for the info! I think I need some more investigation on this.

**Comment by chris-jackson7** [comment]
Sounds good! Thanks for getting to this so quick. I was worried it my be a while lol. Also big fan of your work

**Comment by HenryRLee** [comment]
Sorry I had been busy on my work in the past few days.

It appears that it's the Python3.7 interpreter doesn't recognize the syntax `list[int]` as an argument hint. I've tried this in Python3.7:
```
$ python3.7
Python 3.7.16 (default, Dec  6 2022, 19:08:09)
[GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> def hash_quinary(quinary: list[int], num_cards: int) -> int: return 0
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'type' object is not subscriptable
```

Looks like this is ok:
```
>>> def hash_quinary(quinary: [int], num_cards: int) -> int: return 0
...
>>>
```

I'll fix it in the pip package.

**Comment by HenryRLee** [comment]
Hi @chris-jackson7, I fixed the code in #75 and updated the pypi package. Can you upgrade the package and see if it works?

**Comment by chris-jackson7** [comment]
Yea just imported it without issue! Thanks for looking into this!
